powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/OperationsController.java

```diff
@@ -59,10 +63,13 @@ public OperationsController(OperationServiceBehavior service) {
      * @throws Exception In case the service throws exception.
      */
     @PostMapping("/create")
-    public ObjectResponse<OperationDetailResponse> createOperation(@RequestBody ObjectRequest<OperationCreateRequest> request) throws Exception {
-        logger.info("OperationCreateRequest received: {}", request);
+    public ObjectResponse<OperationDetailResponse> createOperation(@Valid @RequestBody ObjectRequest<OperationCreateRequest> request) throws Exception {
+        final OperationCreateRequest req = request.getRequestObject();
+        logger.info("action: createOperation, state: initiated, userId: {}, applications: {}, templateName: {}", req.getUserId(), req.getApplications(), req.getTemplateName());
+        logger.debug("action: createOperation, state: initiated, request: {}", request);
         final ObjectResponse<OperationDetailResponse> response = new ObjectResponse<>(service.createOperation(request.getRequestObject()));
-        logger.info("OperationCreateRequest succeeded: {}", response);
+        logger.info("action: createOperation, state: succeeded");
+        logger.debug("action: createOperation, state: succeeded, response: {}", response);
```

Comment:
Great, I vote for keeping `debug` level to be able to increase log detail if needed.